rotkehlchen/tests/utils/database.py

```diff
@@ -1,9 +1,9 @@
 import os
+import random
```

Comment:
@yabirgb  Make sure your editor uses isort and same version as pinned in requirements. This is my editor using isort and correcting location.